lib/router/index.js

```diff
@@ -172,6 +173,14 @@ proto.handle = function handle(req, res, out) {
   req.baseUrl = parentUrl;
   req.originalUrl = req.originalUrl || req.url;
 
+  if(self.maxURLLength && req.url && req.url.length > self.maxURLLength) {
+    var uriError = new URIError();
```

Comment:
Not sure if using `URIError` is semantically correct or if it is supported in all our versions for v4? Just want to confirm one way or another, but I am not personally opposed if it meets both of those.

Reply:
Actually I wonder if we use `http-errors` in here? Should we?

Reply:
https://github.com/jshttp/http-errors?tab=readme-ov-file#list-of-all-constructors

Reply:
and we do import it already `"http-errors": "2.0.0",`

Reply:
fixed